src/containers/Statistics/MessagesNumber/MessagesNumber.js

```diff
@@ -834,32 +835,38 @@ const MessagesNumber = (props) =>{
       border: "1px solid black"
     }}
   >
-  <div className='d-flex mb-2'>
-    <a href="/addStation" className="btn btn-primary">Add Station</a>
```

Comment:
Naravoucenije: kada navigiramo unutar aplikacije onda izbegavamo klasican html <a> element jer on rerefreshuje ceo frontend, nego koristimo react-router funkcionalnosti koje to rutiranje obave na samom frontendu na klijentskom racunaru, a bekendu se frontend samo obraca da dobije podatke koji su mu potrebni (za tabele, grafike, liste, forme itd.).